src/Assignment19/AdvancedLinq/SupplierManager.cs

```diff
@@ -0,0 +1,73 @@
+﻿// <copyright file="SupplierManager.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace AdvancedLinq
+{
+    /// <summary>
+    /// SupplierManager manages the list of suppliers.
+    /// </summary>
+    public class SupplierManager
+    {
+        /// <summary>
+        /// Gets or sets the list of suppliers
+        /// </summary>
+        /// <value>List of Suppliers</value>
+        public static List<Supplier> SuppliersList { get; set; } = new List<Supplier>();
+
+        /// <summary>
+        /// Adds default suppliers to the SuppliersList.
+        /// </summary>
+        public void InitializeWithDefaultSuppliers()
+        {
+            Supplier firstSupplier = new Supplier();
+            firstSupplier.SupplierId = 1;
```

Comment:
Use Object initializer here also

Reply:
Object is initialized with its properties